blaze-client/src/main/scala/org/http4s/blaze/client/Http1Connection.scala

```diff
@@ -405,7 +405,12 @@ private final class Http1Connection[F[_]](
           status = status,
           httpVersion = httpVersion,
           headers = headers,
-          entity = Entity(body.interruptWhen(idleTimeoutS)),
+          entity = entity match {
+            case Entity.Default(body, length) =>
+              Entity[F](body.interruptWhen(idleTimeoutS), length)
+            case _ =>
```

Comment:
Even it's hard to imagine we will add another subtype of `Entity`, nevertheless, I'm sure that using a wildcard on pattern matching isn't good practice. Verbosity isn't always bad.
```suggestion
            case Entity.Strict(_) | Entity.Empty =>
```

Reply:
Good point, thanks.